Makefile

```diff
@@ -121,3 +121,10 @@ devtools:
 	@type "npm" 2> /dev/null || echo 'Please install node.js and npm'
 	@type "solc" 2> /dev/null || echo 'Please install solc'
 	@type "protoc" 2> /dev/null || echo 'Please install protoc'
+
+# The devtoolsForTest target installs tools required for 'go-bindata'.
+# You need to put $BIN (or $GOPATH/bin) in your PATH to use 'go-bindata'.
+# If you upload bindata file, you must update under go-bindata version same wtih yours.
+
+devtools-for-test:
+	env GOFLAGS= GOBIN= go get -u github.com/kevinburke/go-bindata/go-bindata@v3.22.0
```

Comment:
If we need to install this package all the time, how about adding this package to `go.mod`?

Reply:
Thanks JK!
go-bindata run on command...
I think adding dependency in go.mod doesn't works in command..

Reply:
Oh, I think I misunderstood for the test. Sorry for confusion.